pkg/ebpf/gadgets/http/types/types.go

```diff
@@ -0,0 +1,90 @@
+package types
+
+import (
+	"fmt"
+	"net"
+	"net/http"
+
+	"github.com/inspektor-gadget/inspektor-gadget/pkg/columns"
+	eventtypes "github.com/inspektor-gadget/inspektor-gadget/pkg/types"
+	"github.com/kubescape/storage/pkg/apis/softwarecomposition/consts"
+)
+
+var ConsistentHeaders = []string{
+	"Accept-Encoding",
+	"Accept-Language",
+	"Connection",
+	"Host",
+	"Upgrade-Insecure-Requests",
+}
+
+var writeSyscalls = map[string]bool{
```

Comment:
Use the map type we use in all of the node-agent.

Reply:
Which is?

Reply:
`"github.com/goradd/maps"`

Reply:
Does it really matter? both these maps are like consts